src/Validation/Validation.php

```diff
@@ -1020,6 +1022,45 @@ public static function uploadedFile($file, array $options = [])
         return true;
     }
 
+/**
+ * Validates a geographic coordinate.
+ *
+ * Supported formats are right now:
+ *
+ * - <latitude>, <longitude> Example: -25.274398, 133.775136
+ *
+ * ### Options
+ *
+ * - `type` - A string of the coordinate format, right now only `longLat`.
+ * - `longLat` - By default `both`, can be `long` and `lat` as well to validate
+ *   only a part of the coordinate.
+ *
+ * @param string Geographic location as string
+ * @param array $options Options for the validation logic.
+ * @return boolean
+ */
+    public static function geoCoordinate($value, array $options = [])
```

Comment:
I agree with @Marlinc. Better to have separate methods and then a wrapper to validate string of format `<lat>, <long>`.